docs/auth-modes.md

```diff
@@ -228,6 +228,42 @@ Note: the ```value``` attribute value for all options is customizable by connect
 })
 ```
 
+### Database impersonation using embedded credentials (DelegationUID)
+
+This applies to databases which support a user that can delegate requests from other user(s). Databases support this by enabling the client to pass a DelegationUID to the server.
+In the case of Tableau server, you can pass the identity of the logged-in user on Tableau Server as the DelegationUID. Tableau would then pass that property to the database using driver properties and authentication is taken care of by the database. This community [article](https://community.tableau.com/docs/DOC-11137) gives more information on Database Impersonation using Embedded credentials.   
+
+For this to work on Tableau Server, user should select the option "Impersonate using embedded password" while publishing the workbook from  Tableau Desktop. Below is an example for how to pass DelegationUID in a JDBC plugin. Please note that some JDBC drivers may have a different name for this property so take a look at driver documentation for the appropriate property name.
+ 
+ ```javascript
+    // Connection properties
+    function isEmpty(str) {
+        return (!str || 0 === str.length); 
+    }
+
+    var props = {};
+    props["UID"] = attr[connectionHelper.attributeUsername];
+    props["PWD"] = attr[connectionHelper.attributePassword];
+   
+    if (attr[connectionHelper.attributeTableauServerAuthMode] == connectionHelper.valueAuthModeDBImpersonate) {
+        var str = attr[connectionHelper.attributeTableauServerUser];
+        
+        if (!isEmpty(str)){
+            props["DelegationUID"] = str;
+        }
+    }
+ ```
+ 
+ Also add CAP_AUTH_DB_IMPERSONATE to the list of capabilities in plugin manifest so that the option "Impersonate using embedded password" shows up in the publish dialog.
+ ```xml 
+   // manifest.xml 
+    <customizations>
+      <customization name="CAP_AUTH_DB_IMPERSONATE" value="yes"/>
+    </customizations>
+ ```
+ 
+ Impala delegation sample link(TBA)  
```

Comment:
Don't forget to update this

Reply:
sent another PR for the sample -  https://github.com/tableau/connector-plugin-sdk/pull/522

Reply:
I am thinking the link would be something in the tree of master branch, and not dev-2020-3 branch. So I guess I should link to this path https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins/db_impersonation which does not exist yet, but will exist once the merge from 2020-3 to master takes place